examples/docs_snippets/docs_snippets/concepts/partitions_schedules_sensors/schedules/schedules.py

```diff
@@ -24,7 +24,10 @@ def configurable_job():
 @schedule(job=configurable_job, cron_schedule="0 0 * * *")
 def configurable_job_schedule(context: ScheduleEvaluationContext):
     scheduled_date = context.scheduled_execution_time.strftime("%Y-%m-%d")
-    return {"ops": {"configurable_op": {"config": {"scheduled_date": scheduled_date}}}}
+    return RunRequest(
```

Comment:
should there be run_key as well?

Reply:
We don't need a run_key if we only have one run request, since requests are only idempotent for a given schedule clock tick (as opposed to sensors, which are idempotent across sensor ticks).

If we have a generator with multiple run requests, we do have a very clear error message telling users to add run keys.